ext/node/ops/require.rs

```diff
@@ -401,6 +402,53 @@ pub fn op_require_real_path<
   Ok(canonicalized_path.to_string_lossy().into_owned())
 }
 
+#[op2]
+#[string]
+pub fn op_require_fallback_resolve<
+  TInNpmPackageChecker: InNpmPackageChecker + 'static,
+  TNpmPackageFolderResolver: NpmPackageFolderResolver + 'static,
+  TSys: ExtNodeSys + 'static,
+>(
+  state: &mut OpState,
+  #[string] request: String,
+  #[string] parent_filename: Option<String>,
+) -> Result<String, RequireError> {
+  let resolver = state.borrow::<crate::DenoResolverRc<
+    TInNpmPackageChecker,
+    TNpmPackageFolderResolver,
+    TSys,
+  >>();
+  let graph_container = state.borrow::<crate::GraphContainerWrapper>();
```

Comment:
It's probably too much to expose all the way down here and also it creates a dependency on deno_graph for Deploy as well as `deno compile`. Probably we should have a trait that gets passed down here and then the CLI can implement this (maybe just put it as a method on `NodeRequireLoader`)